dss-core/src/main/java/io/github/ztkmkoo/dss/core/actor/blocking/jdbc/DssDatabaseServiceActor.java

```diff
@@ -0,0 +1,69 @@
+package io.github.ztkmkoo.dss.core.actor.blocking.jdbc;
+
+import com.healthmarketscience.sqlbuilder.BinaryCondition;
+import com.healthmarketscience.sqlbuilder.InsertQuery;
+import com.healthmarketscience.sqlbuilder.InsertSelectQuery;
+import com.healthmarketscience.sqlbuilder.SelectQuery;
+
+import akka.actor.typed.Behavior;
+import akka.actor.typed.javadsl.ActorContext;
+import akka.actor.typed.javadsl.Behaviors;
+import io.github.ztkmkoo.dss.core.message.blocking.jdbc.DssJdbcCommand;
+
+
+public class DssDatabaseServiceActor implements DssDatabaseService<DssJdbcCommand, DssJdbcCommand> {
+	
+	private final ActorContext<DssJdbcCommand> context;
+	
+	private DssDatabaseServiceActor(ActorContext<DssJdbcCommand> context) {
+		this.context = context;
+	}
+	
+	private Behavior<DssJdbcCommand> dssDatabaseServiceActor() {
+		return Behaviors
+				.receive(DssJdbcCommand.class)
+				.onMessage(DssJdbcCommand.Select.class, this::select)
+				.build();
+	}
+	
+    public static Behavior<DssJdbcCommand> create() {
+        return Behaviors.setup(context -> new DssDatabaseServiceActor(context).dssDatabaseServiceActor());
+    }
```

Comment:
Hi, I think this static method goes to top. 
Ref https://github.com/ztkmkoo/dss/wiki/Code-Style(KR).

Reply:
Oh you're right! thank you :)